rasa/core/brokers/pika.py

```diff
@@ -221,38 +219,41 @@ def close_pika_connection(connection: "Connection") -> None:
 
 
 class PikaEventBroker(EventBroker):
+    """Pika-based event broker for publishing messages to RabbitMQ.
+
+    Attributes:
+        host: Pika host.
+        username: Username for authentication with Pika host.
+        password: Password for authentication with Pika host.
+        port: port of the Pika host.
+        queues: Pika queues to declare and publish to.
+        should_keep_unpublished_messages: Whether or not the event broker should
+            maintain a queue of unpublished messages to be published later in
+            case of errors.
+        raise_on_failure: Whether to raise an exception if publishing fails. If
+            `False`, keep retrying.
+        log_level: Logging level.
+
+    """
+
     def __init__(
         self,
         host: Text,
         username: Text,
         password: Text,
         port: Union[int, Text] = 5672,
-        queue: Text = "rasa_core_events",
+        queue: Union[List[Text], Tuple[Text], Text, None] = None,
+        queues: Union[List[Text], Tuple[Text], Text, None] = ("rasa_core_events",),
```

Comment:
Alternatively, keep it clean and `pop` `queue` from `kwargs` in case `queues` is empty?

Reply:
optional: Btw: I recently read this on `Union[WhatEver, None]` (http://google.github.io/styleguide/pyguide.html#3195-nonetype)

Reply:
@wochinge 
> optional: Btw: I recently read this on Union[WhatEver, None]

what exactly do you mean? It's already typed as `Union[..., ..., None]`

Reply:
Ah, I skipped that part that `Union[..., ..., None]` is fine if there are more than 2 types